pkg/ui/static/js/graph.js

```diff
@@ -32,11 +32,18 @@ Prometheus.Graph.stepValues = [
   "1w", "2w", "4w", "8w", "1y", "2y"
 ];
 
-Prometheus.Graph.numGraphs = 0;
+// Get id of last panel and increment.
+let num = 0;
+Object.entries(localStorage).map(([key]) => {
+  if (key.includes("enable-dedup") && num < parseInt(key[key.length - 1], 10)) {
+    num = parseInt(key[key.length - 1], 10);
+  }
+});
+Prometheus.Graph.numGraphs = num;
 
 Prometheus.Graph.prototype.initialize = function() {
   var self = this;
-  self.id = Prometheus.Graph.numGraphs++;
+  self.id = ++Prometheus.Graph.numGraphs;
```

Comment:
```suggestion
  self.id = Prometheus.Graph.numGraphs++;
```

I think it's better to keep this line same as before and instead change above implementation to
```js
let num = -1;
// code ....
Prometheus.Graph.numGraphs = num+1;
```

If we don't do this, the first panel will start from `1` instead of `0` which was the case before. IMO it's preferable to keep the old behaviour if possible.